### ISSUE TITLE
Instance: The "Submit comment" control's change of state away from `disabled` is not conveyed visually |  WCAG 2.2 Violation - There are instances of controls undergoing a programmatic change of state where the difference in state is not conveyed visually.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#33]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- 

#### Further information:

1. 

### Screenshot

![Screenshot showing the 'Submit comment' (\`.chat-submit-button\') control's in its \`disabled\` and non-disabled states. The control's appearance is identical between the two states.](https://github.com/user-attachments/assets/0a407b4d-ce2c-4bce-b63e-e5800f9ef025)

### Code Reference

```
<button type="submit" class="chat-submit-button" aria-label="Submit comment" style="opacity: 1;">
	<svg width="20" height="20" viewBox="0 0 16 16" fill="currentColor" style="" aria-hidden="true">[…]</svg>
</button>
```
